GoogleDataTransport/GDTCORTests/Unit/Helpers/GDTCORTestPrioritizer.m

```diff
@@ -15,7 +15,7 @@
  */
 
 #import "GDTCORTests/Unit/Helpers/GDTCORTestPrioritizer.h"
-
+#import <GoogleDataTransport/GDTCOREvent.h>
```

Comment:
Imports of different kinds should be grouped in blocks separated by newlines. I'm also not entirely sure that this import is needed.